### DISCUSSION TITLE
Use case for virtual Button (vs. Multi)?

### DISCUSSION THREAD

**Discussion by vonglan** [helgoboss/helgobox#295] [Q&A] [answered]
So far, I have used "Multi" also for button-like values.
Is there any disadvantage, or are they "downward compatible"?

**Comment by helgoboss** [comment, marked as answer]
Can't tell at the top of my head. There are some subtle difference for sure. Better don't use Multis for simple on/off buttons! Only use them for velocity-sensitive buttons (e.g. notes).
